content/en/blog/2024/hardening-the-collector-one.md

```diff
@@ -0,0 +1,134 @@
+---
+title: 'Hardening the Collector #1 - A new default bind address'
+linkTitle: A new default bind address for the Collector
+date: 2024-07-02
+author: '[Pablo Baeyens](https://github.com/mx-psi) (OpenTelemetry, Datadog)'
+# prettier-ignore
+cSpell:ignore: awsfirehose awsproxy awsxray Baeyens jaegerremotesampling loki remotetap sapm signalfx skywalking splunk
+issue: 4760
+sig: Collector SIG
+---
+
+The OpenTelemetry Collector recently went through a security audit sponsored by
+the [CNCF](https://www.cncf.io/). As part of this process we recently published
+a security advisory related to a [DoS vulnerability](/blog/2024/cve-2024-36129/)
+that was fully addressed in v0.102.1.
```

Comment:
Perhaps a link to the changelog line or PR?

Reply:
Added a link to the changelog: 0c360aa2b127f2633a9c50de6c506acffc4612c9